python-avd/schema_tools/metaschema/meta_schema_model.py

```diff
@@ -214,6 +217,15 @@ def _generate_yaml_lines(self, target_table: str | None = None) -> Generator[Yam
         # Using the Type of yaml line generator set in the subclass attribute _yaml_line_generator
         yield from self._yaml_line_generator().generate_yaml_lines(schema=self, target_table=target_table)
 
+    def _generate_class_src(self, class_name: str | None = None) -> SrcData:
+        """
+        Returns one instance of "Src" to be used for generating pydantic models of the schemas.
```

Comment:
is it really pydantic?
```suggestion
        Returns one instance of "Src" to be used for generating pydantic models of the schemas.
```

Reply:
nope

Reply:
fixed